src/helpers/classNames.ts

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
A light drop-in replacement for the commonly used classnames module. It is faster and has the same api.